sig-usability/README.md

```diff
@@ -0,0 +1,52 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Usability Special Interest Group
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes project is maximally usable by users of a variety of different backgrounds and abilities. Examples of efforts include internationalization and accessibility of documentation.
+
+The [charter](charter.md) defines the scope and governance of the Usability Special Interest Group.
+
+## Meetings
+* Regular SIG Meeting: [TBDs at TBD PT (Pacific Time)](tbd) (every third week). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=TBD&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](tbd).
+  * [Meeting recordings](tbd).
+
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+
+* Rajakavitha Kodhandapani (**[@Rajakavitha1](https://github.com/Rajakavitha1)**), unknown
+* Gaby Moreno (**[@gaby-moreno](https://github.com/gaby-moreno)**), IBM
+* Tasha Drew (**[@tashimi](https://github.com/tashimi)**), VMWare
+* Himanshu Pandey (**[@unknown](https://github.com/unknown)**), unknown
```

Comment:
@brendandburns https://github.com/hpandeycodeit
Is there any doc that lists down Roles and responsibilities for a Chair?

Reply:
See the notes here:

https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair